internal/controller/standalone_pgadmin/pod.go

```diff
@@ -303,17 +319,21 @@ loadServerCommand
 	var reloadScript = `
 exec {fd}<> <(:)
 while read -r -t 5 -u "${fd}" || true; do
-	if [ "${cluster_file}" -nt "/proc/self/fd/${fd}" ] && loadServerCommand
-	then
-		exec {fd}>&- && exec {fd}<> <(:)
-		stat --format='Loaded shared servers dated %y' "${cluster_file}"
-	fi
-	if [ ! -d /proc/$(cat $PGADMIN4_PIDFILE) ]
-	then
-		pgadmin4 &
-		echo $! > $PGADMIN4_PIDFILE
-		echo "Restarting pgAdmin4"
-	fi
+    if [ "${cluster_file}" -nt "/proc/self/fd/${fd}" ] && loadServerCommand
+    then
+        exec {fd}>&- && exec {fd}<> <(:)
+        stat --format='Loaded shared servers dated %y' "${cluster_file}"
+    fi
+    if [ ! -d /proc/$(cat $PGADMIN4_PIDFILE) ]
+    then
+        if [ $APP_RELEASE -eq 7 ]; then
+            ` + startCommandV7 + `
```

Comment:
I like having the string added in inline over passing it in through a print :heart: I think it makes the bash more readable